ocs_ci/ocs/resources/pod.py

```diff
@@ -440,23 +440,27 @@ def exec_cmd_on_node(self, server, authkey, cmd, user=None):
         cmd = f'ssh -i {authkey} -o "StrictHostKeyChecking no" {user}@{server} {cmd}'
         self.exec_cmd_on_pod(cmd, out_yaml_format=False)
 
-    def get_memory(self):
+    def get_memory(self, container_name):
```

Comment:
Found 1 reference in 1 file
ocs_ci/ocs/cluster.py
osd_pod_mem_size_str = osd_pods[0].get_memory().get("osd")